sdks/java/io/google-cloud-platform/expansion-service/build.gradle

```diff
@@ -35,5 +35,13 @@ dependencies {
     permitUnusedDeclared project(":sdks:java:expansion-service") // BEAM-11761
     implementation project(":sdks:java:io:google-cloud-platform")
     permitUnusedDeclared project(":sdks:java:io:google-cloud-platform") // BEAM-11761
+    implementation project(":sdks:java:extensions:schemaio-expansion-service")
+    permitUnusedDeclared project(":sdks:java:extensions:schemaio-expansion-service") // BEAM-11761
```

Comment:
schemaio-expansion-service actually is just that class already, it just pulls in other dependencies and publishes a shaded jar. With this change the only other use of the schemaio-expansion-service shaded jar is for jdbc:

https://github.com/apache/beam/blob/1ef5056383195eb25374f8a2815d6fffb1b5e823/sdks/go/pkg/beam/io/xlang/jdbcio/jdbc.go#L64
https://github.com/apache/beam/blob/8578c9a9352a8a5c9621647e9c585321f9761dc5/sdks/python/apache_beam/io/jdbc.py#L98

We could instead make a jdbc expansion service, and stop creating a shaded jar for schemaio-expansion-service.

Reply:
Won't SchemaIO used by other non-GCP connectors in the future ?
cc: @pabloem

Reply:
Yes but they could publish their own expansion services in the same way right?

Reply:
Ideally we should maintain a grouped set of limited expansion service jars. Each expansion service jar is a shaded jar and can be large.

Reply:
Ok fair point, but regardless I think what we're both proposing is aligned. I'm just suggesting that instead of making a new extension to contain [`ExternalSchemaIOTransformRegistrar.java`](https://github.com/apache/beam/blob/master/sdks/java/extensions/schemaio-expansion-service/src/main/java/org/apache/beam/sdk/extensions/schemaio/expansion/ExternalSchemaIOTransformRegistrar.java) we can leave it where it is, and move away from publishing a shaded expansion service jar with it.

Reply:
Yeah, makes sense. Thanks.